### ISSUE TITLE
validator crashes with an error

### ISSUE THREAD

**Issue by paxarpp** [chantouchsek/validatorjs#423]
"@chantouchsek/validatorjs": "^1.1.5"

if the validation rule is written incompletely, then when the rule is triggered, the validator crashes with an error.

correct rules (work):
```
validationRules: {
        inputFileFormat: 'ref_required',
        dslId: 'required',
        specificationId: `required_if:resultTransform,${DSL_RESULT_TRANSFORM.offer}`,
      },
      validationMessages: {
        'ref_required.inputFileFormat': 'specify value' as const,
        'required.dslId': 'specify value' as const,
        'required_if.specificationId': 'specify value' as const,
      },
```
example rules (crash validator):
```
validationRules: {
        inputFileFormat: 'ref_required',
        dslId: 'required',
        specificationId: `required_if`, // <--
      },
      validationMessages: {
        'ref_required.inputFileFormat': 'specify value' as const,
        'required.dslId': 'specify value' as const,
        'required_if.specificationId': 'specify value' as const,
      },
```

**Comment by paxarpp** [comment]
<img width="1071" alt="Снимок экрана 2023-04-05 в 23 37 45" src="https://user-images.githubusercontent.com/34523588/230206470-f1383aa0-f8f3-4f89-8f17-596696d51dd4.png">

**Comment by chantouchsek** [comment]
So what do u expect it to handle?

**Comment by chantouchsek** [comment]
Ok, let me check it.
